recipes/aom/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.29.1"
+
+class Aom(ConanFile):
+    name = "aom"
```

Comment:
This name might be wrong

`aom` is just short for Alliance For Open Media, but it isn't the canonical name of the AV1 library. AOM will publish further codecs for sure. 

The question is if they will maintain several codecs in one single repository and should those all be available in one single Conan package?

The canonical name for the AV1 library is `libaom-av1`, see e.g.

https://trac.ffmpeg.org/wiki/Encode/AV1

https://aomedia.googlesource.com/aom/+/refs/heads/master/CMakeLists.txt#255

Reply:
OTH, debian uses just [libaom](https://packages.debian.org/sid/libaom-dev)
I agree the name `libaom` is confusing, it's like naming h.265 as just mpeg

Reply:
Yes, I thought it is a little strange to name the upstream project as aom. [Fedora](https://src.fedoraproject.org/rpms/aom) and [Debian](https://tracker.debian.org/pkg/aom) have this package named as `aom`. I have not found any 'libav1' mentions in upstream sources too as well in cmakelists.txt

I think the library and major packagers will keep the name 'aom' for the foreseeable future (is is 1.0). If AOM will publish a new codec, it probably will have a different name, like 'aom-av2'.

Reply:
for conan, let's use `libaom-av1`, as it's most precise thing
AOM itself is confusing, as it specifies an organization name
AV1 itself is confusing, as it specifies codec (standard/format), which could be implemented by different libraries

Reply:
Btw, homebrew, arch, FreeBSD all have `aom`.

I am not against `libaom-av1`, it makes sense. If divergence with major Linux distributions is not critical, let's use it.

Reply:
Done!

Failure with builders (`Current Conan version (1.28.1) does not satisfy the defined one (>=1.29.1).`) is because of very old (a half year) Conan version on Windows builder

Reply:
@ericLemanissier another for #4221